skins/oasis/css/core/ads.scss

```diff
@@ -122,6 +122,26 @@ $taboola-small-breakpoint: "#{$screen} and (max-width: 420px)";
 	}
 }
 
+#INCONTENT_WRAPPER {
+  text-align: center;
+}
+
+#INCONTENT_LEADERBOARD {
+  display: inline-block;
+}
+
+#INCONTENT_LEADERBOARD:before {
+  color: #656d77;
```

Comment:
Where does this color comes from? Is there a variable we can re-use here? How does it look on light and dark wikis? If it's just copied from fandom we might want to assign it to a variable and mention in the ticket it's now constant.